config/crd/bases/nac.oadp.openshift.io_nonadminbackups.yaml

```diff
@@ -503,6 +503,22 @@ spec:
                       type: string
                     type: array
                 type: object
+              logLevel:
```

Comment:
where we are using this?

Reply:
Because I want to have logger.Info and logger.V(1).Info which is debug and in production we will be using Info, but with option to run as debug to get much more insight if something goes wrong.

Reply:
Here is the output of DEBUG, normally just INFO should be in the terminal:

```shell
2024-04-15T22:27:35+02:00	DEBUG	NonAdminBackupPredicate: Received Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:35+02:00	DEBUG	NonAdminBackupPredicate: Rejecting Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	>>> Reconcile NonAdminBackup - loop start	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Phase is already up to date	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Condition is already set to: BackupAccepted	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	INFO	No backup found	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "Name": "nab-nacproject-67bd682a88f045"}
2024-04-15T22:27:45+02:00	INFO	VeleroBackup successfully created	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "Name": "nab-nacproject-67bd682a88f045"}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Phase is already up to date	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Condition is already set to: BackupAccepted	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Condition to: BackupQueued	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Condition Reason to: backup_scheduled	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Condition Message to: Created Velero Backup object	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	VeleroBackupPredicate: Received Create event	{"VeleroBackupPredicate": {"name":"nab-nacproject-67bd682a88f045","namespace":"openshift-adp"}}
2024-04-15T22:27:45+02:00	DEBUG	Received Create VeleroBackupHandler	{"VeleroBackupHandler": {"name":"nab-nacproject-67bd682a88f045","namespace":"openshift-adp"}}
2024-04-15T22:27:45+02:00	DEBUG	VeleroBackupPredicate: Received Update event	{"VeleroBackupPredicate": {"name":"nab-nacproject-67bd682a88f045","namespace":"openshift-adp"}}
2024-04-15T22:27:45+02:00	DEBUG	Received Update VeleroBackupHandler	{"VeleroBackupHandler": {"name":"nab-nacproject-67bd682a88f045","namespace":"openshift-adp"}}
2024-04-15T22:27:45+02:00	DEBUG	>>> Reconcile NonAdminBackup - loop end	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "89397ab2-4c5f-405e-a130-991dcd4b786b", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackupPredicate: Received Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackupPredicate: Rejecting Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	>>> Reconcile NonAdminBackup - loop start	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "d67e8c79-4a85-4615-aa43-2bafaf9202d6", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Phase is already up to date	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "d67e8c79-4a85-4615-aa43-2bafaf9202d6", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Condition is already set to: BackupAccepted	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "d67e8c79-4a85-4615-aa43-2bafaf9202d6", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	INFO	Backup already exists, updating NonAdminBackup status	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "d67e8c79-4a85-4615-aa43-2bafaf9202d6", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "Name": "nab-nacproject-67bd682a88f045"}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup BackupSpec and BackupStatus - request to update	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "d67e8c79-4a85-4615-aa43-2bafaf9202d6", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup BackupStatus - updated	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "d67e8c79-4a85-4615-aa43-2bafaf9202d6", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackupPredicate: Received Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackupPredicate: Rejecting Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackupPredicate: Received Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackupPredicate: Accepted Update event - generation change	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup BackupSpec - updated	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "d67e8c79-4a85-4615-aa43-2bafaf9202d6", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup CR - Rqueue after Status Update	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "d67e8c79-4a85-4615-aa43-2bafaf9202d6", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	>>> Reconcile NonAdminBackup - loop start	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "690a1ae9-af0f-4c34-be5b-0d8bec00fd81", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Phase is already up to date	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "690a1ae9-af0f-4c34-be5b-0d8bec00fd81", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Condition is already set to: BackupAccepted	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "690a1ae9-af0f-4c34-be5b-0d8bec00fd81", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	INFO	Backup already exists, updating NonAdminBackup status	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "690a1ae9-af0f-4c34-be5b-0d8bec00fd81", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "Name": "nab-nacproject-67bd682a88f045"}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup BackupSpec and BackupStatus - request to update	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "690a1ae9-af0f-4c34-be5b-0d8bec00fd81", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup BackupSpec and BackupStatus - nothing to update	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "690a1ae9-af0f-4c34-be5b-0d8bec00fd81", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	VeleroBackupPredicate: Received Update event	{"VeleroBackupPredicate": {"name":"nab-nacproject-67bd682a88f045","namespace":"openshift-adp"}}
2024-04-15T22:27:45+02:00	DEBUG	Received Update VeleroBackupHandler	{"VeleroBackupHandler": {"name":"nab-nacproject-67bd682a88f045","namespace":"openshift-adp"}}
2024-04-15T22:27:45+02:00	DEBUG	>>> Reconcile NonAdminBackup - loop start	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "1d3d5e4a-369c-45b5-b456-7d08de8196bf", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Phase is already up to date	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "1d3d5e4a-369c-45b5-b456-7d08de8196bf", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup Condition is already set to: BackupAccepted	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "1d3d5e4a-369c-45b5-b456-7d08de8196bf", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	INFO	Backup already exists, updating NonAdminBackup status	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "1d3d5e4a-369c-45b5-b456-7d08de8196bf", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "Name": "nab-nacproject-67bd682a88f045"}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup BackupSpec and BackupStatus - request to update	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "1d3d5e4a-369c-45b5-b456-7d08de8196bf", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup BackupStatus - updated	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "1d3d5e4a-369c-45b5-b456-7d08de8196bf", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup BackupSpec - up to date	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "1d3d5e4a-369c-45b5-b456-7d08de8196bf", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackup CR - Rqueue after Status Update	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "1d3d5e4a-369c-45b5-b456-7d08de8196bf", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackupPredicate: Received Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:45+02:00	DEBUG	NonAdminBackupPredicate: Rejecting Update event	{"NonAdminBackupPredicate": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:55+02:00	DEBUG	>>> Reconcile NonAdminBackup - loop start	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "fac26a21-3059-4928-a3c4-9f568454df68", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:55+02:00	DEBUG	NonAdminBackup Phase is already up to date	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "fac26a21-3059-4928-a3c4-9f568454df68", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:55+02:00	DEBUG	NonAdminBackup Condition is already set to: BackupAccepted	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "fac26a21-3059-4928-a3c4-9f568454df68", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:55+02:00	INFO	Backup already exists, updating NonAdminBackup status	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "fac26a21-3059-4928-a3c4-9f568454df68", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "Name": "nab-nacproject-67bd682a88f045"}
2024-04-15T22:27:55+02:00	DEBUG	NonAdminBackup BackupSpec and BackupStatus - request to update	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "fac26a21-3059-4928-a3c4-9f568454df68", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}
2024-04-15T22:27:55+02:00	DEBUG	NonAdminBackup BackupSpec and BackupStatus - nothing to update	{"controller": "nonadminbackup", "controllerGroup": "nac.oadp.openshift.io", "controllerKind": "NonAdminBackup", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}, "namespace": "nacproject", "name": "testnonadminbackup", "reconcileID": "fac26a21-3059-4928-a3c4-9f568454df68", "NonAdminBackup": {"name":"testnonadminbackup","namespace":"nacproject"}}


```